include/wx/msw/private.h

```diff
@@ -91,12 +91,13 @@ WXDLLIMPEXP_BASE void wxSetInstance(HINSTANCE hInst);
 // Return the height of a native text control corresponding to the given
 // character height (as returned by GetCharHeight() or wxGetCharSize()).
 //
-// The wxWindow parameter must be valid and used for getting the DPI.
-inline int wxGetEditHeightFromCharHeight(int cy, const wxWindow* w)
+// The wxWindow parameter is not used.
```

Comment:
```suggestion
// The wxWindow parameter is currently not used but should still be valid.
```
Just in case we decided to use it again in the future.